packer/scripts/install-lifecycled.sh

```diff
@@ -13,7 +13,7 @@ sudo curl -Lf -o /etc/init/lifecycled.conf \
 cat << EOF | sudo tee /usr/bin/buildkite-lifecycle-handler
 #!/bin/sh -eu
 echo "stopping buildkite-agent gracefully"
-service buildkite-agent stop
+pgrep buildkite-agent | xargs kill
```

Comment:
The reason for this is because there's no longer a buildkite-agent system service at all…